Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_006.GitHubPat.sarif

```diff
@@ -83,11 +83,10 @@
             }
           ],
           "fingerprints": {
-            "AssetFingerprint/v1": "[platform=GitHub]",
-            "ValidationFingerprint/v1": "[secret=ff34885a8624460a855540c6592698d2f1812843]",
-            "ValidationFingerprintHash/v1": "25799b447c11c0dec40e393f9b0349fbdf11bbb5bb8c2e6c698dbbd1bbe7cb7c",
-            "AssetFingerprint/v2": "{\"platform\":\"GitHub\"}",
-            "ValidationFingerprint/v2": "{\"secret\":\"ff34885a8624460a855540c6592698d2f1812843\"}"
+            "AssetFingerprint/current": "[platform=GitHub]",
```

Comment:
>[platform=GitHub] [](http://example.com/codeflow?start=41&length=17)

this should be json.

this is also related to my other comment saying that you are missing an argument

Reply:
the same applies for validationfingerprint